tests/unit/api/objectHead.js

```diff
@@ -137,6 +137,27 @@ describe('objectHead API', () => {
         });
     });
 
+    it('should return Accept-Ranges header if request header includes' +
```

Comment:
your code changes introduced a total of 4 new paths through the method but you only added one new test. Not a requirement but consider adding tests for all paths. Just my general practice anyway.

Reply:
done; added new unit tests and all the paths should be covered now